config/jobs/kubernetes/test-infra/test-infra-presubmits.yaml

```diff
@@ -65,3 +65,28 @@ presubmits:
         - prow/cluster
     annotations:
       testgrid-create-test-group: 'true'
+  - name: pull-test-infra-prow-checkconfig
+    branches:
+      - master
+    decorate: true
+    cluster: build
+    run_if_changed: '^config/prow/(config|plugins).yaml$'
```

Comment:
We also want to validate changes to job config (any YAML file under https://github.com/kubernetes/test-infra/tree/master/config/jobs)